fastembed/rerank/cross_encoder/text_cross_encoder_base.py

```diff
@@ -32,6 +32,27 @@ def rerank(
             **kwargs: Additional keyword argument to pass to the rerank method.
 
         Yields:
-            Iterable[float]: The scores of reranked the documents.
+            Iterable[float]: The scores of the reranked the documents.
+        """
+        raise NotImplementedError("This method should be overridden by subclasses")
+
+    def rerank_pairs(
+        self,
+        pairs: Iterable[tuple[str, str]],
+        batch_size: int = 64,
+        parallel: Optional[int] = None,
+        **kwargs: Any,
+    ) -> Iterable[float]:
+        """Rerank query-document pairs.
+        Args:
+            pairs Iterable[tuple[str, str]]: Query-document pairs to rerank
```

Comment:
```suggestion
            pairs (Iterable[tuple[str, str]]): Query-document pairs to rerank
```

Otherwise, IDE is complaining about pairs not presented in the docstring